src/mono/wasm/testassets/WasmBasicTestApp/wwwroot/WasmBasicTestApp.lib.module.js

```diff
@@ -0,0 +1,14 @@
+export function onRuntimeConfigLoaded(config) {
```

Comment:
I'm not sure I like this feature, do we need it for something specific ?
It will delay dotnet startup only after user code is loaded, parsed and executed.

Reply:
I see maybe this is for Blazor's `beforeStart`. But I still don't love it. Could we make it opt, in via MSBuild metadata or something?